pkgs/top-level/all-packages.nix

```diff
@@ -11454,7 +11456,7 @@ with pkgs;
       };
     })
     openssl_1_0_2
-    openssl_1_1_0;
+    openssl_1_1_1;
```

Comment:
I think we should not introduce `openssl_1_1_1` attribute, we should rename `openssl_1_1_0` to `openssl_1_1` and use that.

Reply:
Same with `openssl_1_0`?

Reply:
No, it does not have to be changed in this PR, and it may be left as is: AFAIK `openssl_1_0_1` and `openssl_1_0_2` were incompatible.

Reply:
Please don't introduce `openssl_1_0` alias for `openssl_1_0_2` yet.

Reply:
Yes, please use `openssl_1_1` and `openssl_1_0_2`.

Reply:
I don't feel strongly either way, but if future 1.1.x versions are expected to break backwards compatibility to 1.1.1, having that version explicitly makes sense to me.